setup.py

```diff
@@ -14,4 +14,6 @@
     setup(name="pybids",
           version=versioneer.get_version(),
           cmdclass=versioneer.get_cmdclass(),
-          setup_requires=SETUP_REQUIRES)
+          setup_requires=SETUP_REQUIRES,
+          install_requires=['packaging', 'sqlalchemy']
```

Comment:
`install_requires` are defined in `setup.cfg`.

Reply:
@effigies Oh... good! Thanks for the pointer. I'll adjust. How come the CI does not take these into account? It complains because these packages are missing. Adding them to install_requires here solve that, but it seems that it does not see those in the .cfg.

Reply:
@effigies Thinking about it, maybe it is because I overshadowed them with the setup.py. I'll remove it from setup.py and see if it solves this issue at the same time.

Reply:
Yes, if you specify something in `setup.py`, it will override the settings in `setup.cfg`.